subsys/net/lib/config/Kconfig

```diff
@@ -210,6 +210,13 @@ config NET_CONFIG_DHCPV6_REQUEST_PREFIX
 	  When DHCPv6 is enabled this will configure the DHCPv6 client to
 	  request IPv6 prefix from the DHCPv6 server.
 
+config NET_CONFIG_DHCPV6_DUID_MAX_LEN
+	int "The maximum DUID length (not including a type code)"
+	range 1 128
+	default 22
+	help
+	  This will set the available number of bytes for the DUID.
```

Comment:
I think this should be set in `subsys/net/lib/dhcpv6/Kconfig` as the config library is an optional component mainly meant for testing purposes.

Reply:
I moved the config to `subsys/net/lib/dhcpv6/Kconfig`